src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,56 @@
+import classNames from 'classnames';
+import { useState } from 'react';
+
+type Props = {
+  setFilter: (filter?: boolean) => void
+};
+
+enum TypeFilter {
+  'all',
+  'active',
+  'completed',
+}
+
+export const Filter: React.FC<Props> = ({ setFilter }) => {
+  const [typeFilter, setTypeFilter] = useState(TypeFilter[1]);
```

Comment:
Use enum in a right way, please 
```suggestion
  const [typeFilter, setTypeFilter] = useState(FilterType.All);
```